scala/lambdas/entity-event-generator-lambda/src/test/scala/uk/gov/nationalarchives/entityeventgenerator/LambdaSpec.scala

```diff
@@ -27,6 +27,33 @@ class LambdaSpec extends AnyFlatSpec with EitherValues {
     lambdaResult.value should equal(1)
   }
 
+  "handler" should "increment the start argument and send messages if all entities are deleted" in {
```

Comment:
1. To match the test below, maybe it could be "increment the start argument, not update the event datetime and send messages..."
2. Maybe "startAt argument" instead of "start argument"?